src/withAxiom.ts

```diff
@@ -108,6 +108,7 @@ function withAxiomNextMiddleware(handler: NextMiddleware): NextMiddleware {
       ev.waitUntil(log.flush());
```

Comment:
Can we rename this function `withAxiomNextMiddleware => withAxiomNextEdgeFunction`?